mailpoet/views/newsletter/editor.html

```diff
@@ -376,6 +376,14 @@ <h1 style="display:none"><%= __('Newsletter Editor') %></h1>
   var mailpoet_installed_at = '<%= settings.installed_at %>';
 </script>
 
+<% if brand_styles.available %>
+<script type="text/javascript">
+  var mailpoet_brand_styles = {
+    available: true
+  };
+</script>
+<% endif %>
+
```

Comment:
Maybe you could always pass the brand_styles object to JS. If you do so you don't have to later in JS/TS handle situations when global `mailpoet_brand_styles` is not defined.

`var mailpoet_brand_style = <%= json_encode(homepage) %>;`

Reply:
You're right about the TS part, but since we'll probably add more settings, thought not adding the object to the DOM at all when the brand styles are not available would be a better approach. But I'm also fine with your approach as we can optimize later.

Reply:
Aha. I see. It makes sense. With my approach, all the other settings would need to be optional or in an optional nested subobject. Feel free to keep it as is or change it as you decide.